Wikipedia/View Controllers/WebView/NativeSectionTitleOverlays/WMFSectionTitlesViewController.m

```diff
@@ -0,0 +1,231 @@
+//  Created by Monte Hurd on 7/31/15.
+//  Copyright (c) 2015 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
+
+#import "WMFSectionTitlesViewController.h"
+#import "WMFTitleOverlayLabel.h"
+#import "WMFTitleOverlayModel.h"
+#import "NSString+Extras.h"
+#import "UIView+WMFSearchSubviews.h"
+#import <Masonry/Masonry.h>
+
+@interface WMFSectionTitlesViewController ()
+
+@property (nonatomic, strong) NSMutableArray* models;
+@property (nonatomic) NSUInteger indexOfNativeTitleLabelNearestTop;
```

Comment:
I'd get away from naming variables with "…titleLabel" and move to something more agnostic like "sectionHeader". They are really more of a container for the title label and may have other things in there that aren't labels (like an edit button).